How to write tests for gluster-block using Glusto framework

bhumikagoyal opened the pull request:
This doc explains how to write gluster-block test cases using the glusto framework. Some examples included for better understanding.

goyalbhumika committed:
Glusto documentation for gluster-block

This doc explains how to write write gluster-block test cases using the glusto framework. Some examples included for better understanding.

goyalbhumika committed:
Update glusto_framework.md

sankarshanmukhopadhyay commented on docs/glusto_framework.md:
```diff
@@ -0,0 +1,199 @@
+
+## Running the tests
+
+Before writing gluster-block test cases, create 3-4 virtual machines which will serve as clients and servers.
```
Suggestion; adding environment detail for the machine instances required.

sankarshanmukhopadhyay commented on docs/glusto_framework.md:
```diff
@@ -0,0 +1,199 @@
+
+@runs_on decorator is used to define the volume types for test cases. Multiple volume types can be given at the same time.
+
+In the below example, the setUpClass method creates a glusterfs volume of type repicated using the information from the config file.
```
'replicated'

sankarshanmukhopadhyay commented on docs/glusto_framework.md:
```diff
@@ -0,0 +1,199 @@
+@runs_on decorator is used to define the volume types for test cases. Multiple volume types can be given at the same time.
+
+In the below example, the setUpClass method creates a glusterfs volume of type repicated using the information from the config file.
+For this example I am assuming that glusterd and gluster-blockd are already started on the servers but they can be started/restarted in the test cases too as you can see below. 
```
For this example, we assume that the glusterd and gluster-blockd srevices are already running on the servers. However, they can be started/re-started in the tests cases as are shown in examples below.

sankarshanmukhopadhyay reviewed (commented):
Suggest changing the PR title to read "How to write tests for gluster-block using Glusto framework" or, similar.

goyalbhumika committed:
How to write tests for gluster-block using Glusto framework

Changes in this commit
1. Add machine environment details
2.  Fix some grammatical errrors.
3. Change PR title

pkalever commented:
This didn't make into the project, closing this now.
